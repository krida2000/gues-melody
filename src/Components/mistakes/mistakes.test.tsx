import * as React from "react";
import * as renderer from "react-test-renderer";
import Mistakes from "./mistakes";
const noop = () => {
  // do nothing
};

it(`Win screen correctly render`, () => {
  const tree = renderer
    .create(
      <Mistakes
        errorsCount={1}
      />).toJSON();

  expect(tree).toMatchSnapshot();
});
