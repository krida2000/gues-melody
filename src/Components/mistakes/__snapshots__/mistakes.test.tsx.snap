// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Win screen correctly render 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
</div>
`;
